datasets/indonlu/indonlu.py

```diff
@@ -0,0 +1,631 @@
+        ]
+
+    def _generate_examples(self, filepath):
+        """ Yields examples. """
+        csv_file = ["emot", "wrete", "facqa", "casa", "hoasa"]
+        tsv_file = ["smsa"]
+        txt_file = ["terma", "keps"]
+        txt_file_pos = ["posp", "bapos"]
+        txt_file_ner = ["nergrit", "nerp"]
+
+        with open(filepath, encoding="utf-8") as f:
+
+            if self.config.name in csv_file:
+                reader = csv.reader(f, delimiter=",", quotechar='"', quoting=csv.QUOTE_ALL)
+                next(reader)  # skip first row which is header
+
+                for id_, row in enumerate(reader):
+                    if self.config.name == "emot":
+                        label, tweet = row
+                        yield id_, {"tweet": tweet, "label": label}
+                    elif self.config.name == "wrete":
+                        sent_A, sent_B, category, label = row
+                        yield id_, {"sent_A": sent_A, "sent_B": sent_B, "category": category, "label": label}
+                    elif self.config.name == "facqa":
+                        question, passage, seq_label = row
+                        yield id_, {"question": eval(question), "passage": eval(passage), "seq_label": eval(seq_label)}
+                    elif self.config.name == "casa" or self.config.name == "hoasa":
+                        sentence, *labels = row
+                        sentence = {"sentence": sentence}
+                        label = {l: labels[idx] for idx, l in enumerate(self.config.label_column)}
+                        yield id_, {**sentence, **label}
+            elif self.config.name in tsv_file:
+                reader = csv.reader(f, delimiter="\t", quoting=csv.QUOTE_NONE)
+
+                for id_, row in enumerate(reader):
+                    if self.config.name == "smsa":
+                        text, label = row
+                        yield id_, {"text": text, "label": label}
+            elif self.config.name in (txt_file + txt_file_pos + txt_file_ner):
+                id_ = 0
+                tokens = []
+                seq_label = []
+                for line in f:
+                    if len(line.strip()) > 0:
+                        token, label = line[:-1].split("\t")
+                        tokens.append(token)
+                        seq_label.append(label)
+                    else:
+                        if self.config.name in txt_file:
+                            yield id_, {"tokens": tokens, "seq_label": seq_label}
+                        elif self.config.name in txt_file_pos:
+                            yield id_, {"tokens": tokens, "pos_tags": seq_label}
+                        elif self.config.name in txt_file_ner:
+                            yield id_, {"tokens": tokens, "ner_tags": seq_label}
+                        id_ += 1
+                        tokens = []
+                        seq_label = []
```

Comment:
```suggestion
                        seq_label = []
                # add last example
                if tokens:
                    if self.config.name in txt_file:
                        yield id_, {"tokens": tokens, "seq_label": seq_label}
                    elif self.config.name in txt_file_pos:
                        yield id_, {"tokens": tokens, "pos_tags": seq_label}
                    elif self.config.name in txt_file_ner:
                        yield id_, {"tokens": tokens, "ner_tags": seq_label}
```
Maybe there's one example that hasn't been yielded